persistence/sql/migrations/sql/20220901123209000000_recovery_code.up.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE identity_recovery_codes
+(
+    id UUID NOT NULL PRIMARY KEY,
+    code VARCHAR (64) NOT NULL, -- HMACed value of the actual code
+    used bool NOT NULL DEFAULT 'false',
+    used_at timestamp,
+    identity_recovery_address_id UUID,
```

Comment:
> The reason being, that there are recovery flows initiated by the admin that do not require a recovery address.

Same reason here.

And this is needed because a recovery address that is not verified will be marked as `verified` if it was used to recover access to the account.